clients/drcachesim/scheduler/scheduler_impl.cpp

```diff
@@ -2834,13 +2834,13 @@ scheduler_impl_tmpl_t<RecordType, ReaderType>::next_record(output_ordinal_t outp
     outputs_[output].last_record = record;
     record_type_has_tid(record, input->last_record_tid);
     record_type_has_pid(record, input->pid);
-    return finalize_next_record(record, input);
+    return finalize_next_record(record, input, output);
 }
 
 template <typename RecordType, typename ReaderType>
 typename scheduler_tmpl_t<RecordType, ReaderType>::stream_status_t
 scheduler_impl_tmpl_t<RecordType, ReaderType>::finalize_next_record(
-    const RecordType &record, input_info_t *input)
+    const RecordType &record, input_info_t *input, output_ordinal_t output)
```

Comment:
nit: The output is always the first parameter

Reply:
Done